test/core/end2end/tests/grpc_authz.cc

```diff
@@ -641,11 +633,28 @@ static void test_file_watcher_invalid_policy_skip_reload(
                  nullptr, &server_args);
   grpc_authorization_policy_provider_release(provider);
   test_allow_authorized_request(f);
+  gpr_event on_reload_done;
+  gpr_event_init(&on_reload_done);
+  std::function<void(bool contents_changed, absl::Status status)> callback =
+      [&on_reload_done](bool contents_changed, absl::Status status) {
+        if (contents_changed) {
+          GPR_ASSERT(absl::StatusCode::kInvalidArgument == status.code());
+          GPR_ASSERT(
+              status.message().compare("\"name\" field is not present.") == 0);
```

Comment:
Instead of `string_view.compare(expected) == 0`, please just say `string_view == expected`.

Same thing throughout.